examples/CMakeLists.txt

```diff
@@ -147,6 +147,10 @@ if (MFEM_USE_SUNDIALS)
   add_subdirectory(sundials)
 endif()
 
+if(MFEM_USE_CALIPER)
+  add_subdirectory(caliper)
```

Comment:
Creating `examples/caliper` leads to duplications, and will get worse with increasing use of Caliper.

Reply:
Discussed at the weekly PR meeting